README.md

```diff
@@ -137,25 +178,24 @@ protoData {
     )
 }
 ```
-Also, the dependencies on `proto-extension` and `codegen-plugin`
-subprojects should be added:
-```kotlin
-dependencies {
-    // Enable field options extension.
-    api(project(":proto-extension"))
-    // Add module with code generation plugin to ProtoData classpath.
-    protoData(project(":codegen-plugin"))
-}
-```
-### Integration Tests
 
-The negative test-cases are implemented in a specific way.
+# Development
+
+### Prerequisites
 
-Such tests configure and run the build of a test project
-in a separate Gradle process because every negative case raises
-the appropriate error and fails the build.
+This example is implemented using the following technologies and tools:
 
-A negative test-case is accepted when the specific error message 
-is found in the stderr stream of the failed build process.
+1. Java 11
+2. Kotlin 1.9.23
+3. Gradle 7.6
 
-See the `integration-tests` subproject for details.
+### Build Instruction
```

Comment:
"### Building locally"

And we don't follow any title capitalization anymore. That's something considered obsolete, as of (relatively) new Material guidelines. [Sentence case](https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices#fc5c2a78-f4bf-4d42-bdac-42ff80391129) is a designed approach.

Please review other headers for that, as well.